core-plugins/docs/PythonEvaluator-transform.md

```diff
@@ -25,35 +25,40 @@ schema is the same as the input schema.
 
 Example
 -------
-The transform takes records that have a ``'subtotal'`` field, calculates ``'tax'`` and
-``'total'`` fields based on the subtotal, and then returns a record, as a Python dictionary,
-containing those three fields:
+The transform takes records that have a ``'subtotal'`` field, if ``'subtotal'`` is negative, it emits an error,
```

Comment:
There is nothing in the reference section about it though.  I can look at the reference and see an example, but I still don't know what it is.  This document should be good as a document on its own.  Somebody can look at it directly in the ui without reading the tooltip, somebody can look at it in github without deploying it to cdap, etc.